style.css

```diff
@@ -92,3 +107,251 @@ header {
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+.multiporpuse > p {
+  font-family: 'Inter', sans-serif;
+  margin: 12px 0;
+}
+
+.button-grid {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  grid-gap: 0 12px;
+  justify-content: flex-start;
+}
+
+.textgrid {
+  border: 1px solid #8993a4;
+  font-family: 'Inter', sans-serif;
+  padding: 10px 12px;
+  margin: 12px 0;
+}
+
+.seeproject {
+  background-color: #ff6b00;
+  grid-column: 1/3;
+  padding: 12px;
+  color: #fff;
+  letter-spacing: 0.03em;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: bold;
+}
+
+.works-container {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto auto;
+  grid-template-columns: 100%;
+  justify-content: center;
+  width: 100%;
+}
+
+.item {
+  background-image: url(assets/lineargradient.png);
+  background-size: 100% auto;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  margin: 12px 0;
+  width: 100%;
+}
+
+.centering {
+  padding: 24px;
+}
+
+.item > h2 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  margin-top: 84px;
+  color: #fff;
+}
+
+.item > p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.works-buttons {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-gap: 0 12px;
+  justify-content: flex-start;
+  margin-bottom: 4px;
+}
+
+.streched {
+  background-color: #ff6b00;
+  padding: 12px;
+  color: #fff;
+  letter-spacing: 0.03em;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: bold;
+  width: 100%;
+  text-align: center;
+  margin: 12px 0;
+}
+
+.aboutme-container {
+  background: #1c1a19;
+  width: 100%;
+  padding-bottom: 120px;
+}
+
+.aboutme-profile {
+  background-color: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 100%;
+  padding: 120px 24px;
+  background-image: url(assets/about-me-mobile.png);
+  background-position: right 75%;
+  background-repeat: no-repeat;
+}
+
+.aboutme-profile > div {
+  display: flex;
+  flex-direction: column;
+}
+
+.aboutme-profile > div > * {
+  margin: 12px 0;
+}
+
+.aboutme {
+  color: #ff6b00;
+  margin-top: 120px;
+  font-size: 72px;
+  font-weight: normal;
+  font-style: normal;
+  font-family: 'Crete Round', serif;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  order: 0;
+  flex-grow: 0;
+}
+
+.resume {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  background: #ff6b00;
+  padding: 12px;
+}
```

Comment:
- [ ] You should add the button styling according to the figma design: hover, pressed and active. Do this for all the buttons

Reply:
Well resolved this as well✅